src/org/usfirst/frc/team1923/robot/commands/gear/IntakeAlignCommand.java

```diff
@@ -92,9 +93,9 @@ public Drivetrain(Side side) {
         @Override
         public void pidWrite(double output) {
             if (this.side == Side.LEFT) {
-                Robot.driveSubSys.setLeft(output * POWER, CANTalon.TalonControlMode.Voltage);
+                Robot.driveSubSys.setLeft(output * IntakeAlignCommand.this.POWER, CANTalon.TalonControlMode.Voltage);

```

Comment:
Plz check over things manually, there's a ton of weird stuff going on in your changes